docs/philosophy.md

```diff
@@ -2,58 +2,58 @@
 
 ## Style and formatting
 
-The focus of Ansible content creators should be on automation, outcomes and readability, rather than style or formatting. Thus is why we follow the same concepts as other code formatting tools like \[black\](<https://github.com/psf/black>) and \[prettier\](<https://prettier.io>).
+The focus of Ansible content creators should be on automation, outcomes and readability, rather than style or formatting. This is why we follow the same concepts as other code formatting tools like [black](https://github.com/psf/black) and [prettier](https://prettier.io).
 
 Adoption of `ansible-lint` will save time by keeping reviews focused on the quality of the content and less so on the nuances of formatting and style.
 
-As code formatting is not really an art, we can save your project time and effort by applying a standardized code style and formatting.
+As code formatting is not an art, we can save your project time and effort by applying a standardized code style and formatting.
 
 ## Q&A
 
 ### Why does ansible-lint not accept all valid ansible syntax?
 
 `ansible-core` continues to mature while maintaining backward compatibility with early versions. `ansible-lint` has never intended to support the whole historical Ansible language syntax variations, but instead only the best of it.
 
-It supports a broad vocabulary of keywords and styles. Over time, changes in the language have lead to an improved experience for authors and consumers of Ansible content. The rules in `ansible-lint` suggest the use of these patterns.
+It supports a broad vocabulary of keywords and styles. Over time, changes in the language have led to an improved experience for authors and consumers of Ansible content. The rules in `ansible-lint` suggest the use of these patterns.
 
-It is these usage patterns that are written as rules in `ansible-lint`,leading to improved readability of **playbooks, roles**, and **collections**. The linter will always be more restrictive and opinionated regarding what it accepts. It is part of its design. We are not forced to keep the same backwards compatibility level as Ansible, so we can tell people to avoid specific syntax for various reasons, such as being deprecated, unsafe, or hard to maintain.
+It is these usage patterns that are written as rules in `ansible-lint`, leading to improved readability of **playbooks, roles**, and **collections**. The linter will always be more restrictive and opinionated regarding what it accepts. It is part of its design. We are not forced to keep the same backward compatibility level as Ansible, so we can tell people to avoid specific syntax for various reasons, such as being deprecated, unsafe, or hard to maintain.
```

Comment:
```suggestion
It is these usage patterns that are written as rules in `ansible-lint`, leading to improved readability of **playbooks**, **roles**, and **collections**. The linter will always be more restrictive and opinionated regarding what it accepts; these opinionated restrictions are part of its design. We do not have to keep the same backward compatibility level as Ansible, so we can tell people to avoid specific syntax for various reasons, such as being deprecated, unsafe, or hard to maintain.
```